frontend/document/credocument.lua

```diff
@@ -796,7 +797,7 @@ function CreDocument:getFontFace()
 end
 
 function CreDocument:getCurrentPos()
-    return self._document:getCurrentPos()
+    self._document:getCurrentPos()
 end
```

Comment:
I'm pretty sure you want to keep `return`'ing something there :)

Reply:
Yepp, I just searched for that.